### DISCUSSION TITLE
Cannot continue after entering the 2FA code

### DISCUSSION THREAD

**Discussion by sseaan** [dyc3/steamguard-cli#351] [Q&A] [answered]
After using the steamguard setup command, I entered my username and password, and after entering the 2FA code, the following characters appeared:
thread 'main' panicked at src/commands/setup.rs:42:14:
Failed to log in. Account has not been linked.: UnknownEResult(DuplicateRequest)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by dyc3** [comment]
Can you run it with `-v debug` and share the output?

**Reply by sseaan** [reply]
DEBUG - Args { global: GlobalArgs { username: None, password: None, all: false, mafiles_path: None, passkey: None, verbosity: Debug, no_update_check: false, http_proxy: None, proxy_credentials: None, danger_accept_invalid_certs: false }, sub: None, code: CodeCommand { offline: false } }
INFO - reading manifest from C:\Users\sseaa\AppData\Roaming\steamguard-cli/maFiles
DEBUG - loading manifest: "C:\\Users\\sseaa\\AppData\\Roaming\\steamguard-cli/maFiles\\manifest.json"
DEBUG - passkey was revoked from manifest
DEBUG - Performing auto-upgrade...
DEBUG - Manifest is up to date
ERROR - Could not load accounts: Could not find an entry in the manifest for this account. Check your spelling.
ERROR - Could not find an entry in the manifest for this account. Check your spelling.
DEBUG - Checking for updates to dyc3/steamguard-cli v0.12.4
DEBUG - No update available

**Reply by dyc3** [reply]
Sorry, I meant going through setup with `-v debug`. Run `steamguard -v debug setup`

**Reply by sseaan** [reply]
![PixPin_2023-12-08_19-17-46](https://github.com/dyc3/steamguard-cli/assets/77724796/b46bd1fc-7753-4184-96c8-274d6978cb48)

**Reply by dyc3** [reply]
Ok this one looks like a different error than the one you posted before. It looks like you already have steamguard set up somewhere else, and it looks like you didn't enter a 2fa code when prompted.

**Reply by sseaan** [reply]
I tried again, and this is the result
![图片](https://github.com/dyc3/steamguard-cli/assets/77724796/ddd11de5-3a4c-4d80-868e-a3b074a88ffa)

**Reply by dyc3** [reply, marked as answer]
Like I said before (and in the error message), you already have an authenticator set up on your steam account. This is not a steamguard-cli bug. Do you have it set up on your phone already? You can't have it set up on both steamguard-cli and the steam mobile app simultaneously, it's either one or the other.

**Reply by sseaan** [reply]
Thank you very much. The problem has been resolved
